deploy/clowdapp.yaml

```diff
@@ -460,6 +460,19 @@ objects:
     - prefix: insights/compliance/policy_hosts
       query: >-
         SELECT "policy_hosts"."host_id", "policy_hosts"."policy_id" FROM "policy_hosts";
+    - prefix: insights/compliance/failed_rules
+      query: >-
+        SELECT "rules"."ref_id", "accounts"."org_id", COUNT("rule_results"."id") as "failed_count" FROM "rule_results"
```

Comment:
Generally I'd discourage doing aggregations, like COUNT here. Would you mind returning individual records with respective primary keys, timestamps and relevant status/complaint fields?

Reply:
I specifically wanted to avoid this to keep the dump size small enough. I am afraid the data here can reach the edge of what floorist can do and I would like to avoid more expensive dumps...